src/util/codecs.js

```diff
@@ -177,23 +167,16 @@ export const codecsForPlaylist = function(master, media) {
       // video are always separate (and separately specified).
       codecInfo.audioProfile = audioProfileFromDefault(master, mediaAttributes.AUDIO);
     }
-
-    if (!codecInfo.audioProfile) {
-      videojs.log.warn(
-        'Multiple audio tracks present but no audio codec string is specified. ' +
-        'Attempting to use the default audio codec (mp4a.40.2)');
-      codecInfo.audioProfile = defaultCodecs.audioProfile;
-    }
   }
 
   const codecs = {};
 
   if (codecInfo.videoCodec) {
-    codecs.video = `${codecInfo.videoCodec}${codecInfo.videoObjectTypeIndicator}`;
+    codecs.video = translateLegacyCodec(`${codecInfo.videoCodec}${codecInfo.videoObjectTypeIndicator}`);
```

Comment:
We only used `translateLegacyCodec` to translate codecs from this function, seems like we should just use in in here.